tsdb/chunks/head_chunks.go

```diff
@@ -696,7 +721,9 @@ func (cdm *ChunkDiskMapper) Chunk(ref ChunkDiskMapperRef) (chunkenc.Chunk, error
 
 	// Encoding.
 	chkEnc := mmapFile.byteSlice.Range(chkStart, chkStart+ChunkEncodingSize)[0]
-
+	sourceChkEnc := chunkenc.Encoding(chkEnc)
+	// update the chunk encoding by restoring the original encoding, if needed
```

Comment:
Other than the reword, in Prometheus we start comments with capital letters where applicable and end with a full stop.
```suggestion
	// Extract the encoding from the byte. ChunkDiskMapper uses only the last 7 bits for the encoding.
```

Reply:
Done